test/serving.bash

```diff
@@ -73,6 +73,7 @@ function run_knative_serving_e2e_and_conformance_tests {
     --imagetemplate "image-registry.openshift-image-registry.svc:5000/serving-tests/{{.Name}}" || failed=2
   
   # Prevent HPA from scaling to make HA tests more stable
+  local max_replicas=$(oc get hpa activator -n "$SERVING_NAMESPACE" -ojsonpath='{.spec.maxReplicas}')
```

Comment:
Define local separately from invocation - shellcheck will catch this :stuck_out_tongue: . Now, result will be okay even if `oc get ..` fails.

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
We don't do that in this file in other places except for one. But ok

Reply:
We should do it everywhere as it is dangerous.

Btw. `local abc=0` isn't a problem, as well as `local abc="${other:-defaults}"`. Dangerous is only `local abc=$(command-that-might-fail)`